docker/worker/Dockerfile

```diff
@@ -1,21 +1,20 @@
+#!BuildTag: openqa_worker
 FROM opensuse/leap:15.2
 LABEL maintainer Jan Sedlak <jsedlak@redhat.com>, Josef Skladanka <jskladan@redhat.com>, wnereiz <wnereiz@eienteiland.org>, Sergio Lindo Mansilla <slindomansilla@suse.com>
 LABEL version="0.3"
 
-RUN zypper ar -f obs://devel:openQA/openSUSE_Leap_15.2 openQA && \
-    zypper ar -f obs://devel:openQA:Leap:15.2/openSUSE_Leap_15.2 openQA-perl-modules && \
+RUN zypper ar -p 95 -f http://download.opensuse.org/repositories/devel:openQA/openSUSE_Leap_15.2 devel_openQA && \
+    zypper ar -p 90 -f http://download.opensuse.org/repositories/devel:openQA:Leap:15.2/openSUSE_Leap_15.2 devel_openQA_Leap && \
     zypper --gpg-auto-import-keys ref && \
     zypper --non-interactive in ca-certificates-mozilla curl gzip && \
     zypper --non-interactive in openQA-worker qemu-arm qemu-ppc qemu-x86 && \
     zypper --non-interactive in kmod && \
-    (zypper --non-interactive in qemu-ovmf-x86_64 || true) && \
-    (zypper --non-interactive in qemu-uefi-aarch64 || true)
+    zypper --non-interactive in qemu-ovmf-x86_64
```

Comment:
I don't see the removal of aarch64 covered in your git commit message and you also don't install that during runtime. Can you revert this?

Reply:
Mmm yes, you are right, it should be on the on the commit message. Fixing that. The reason to remove this is because OBS cannot build the image with this condition in the Dockerfile and secondly the requisite is to have this working for x86 only as pointed @kalikiana,  then this solves the problem 

> I think we should focus on getting a working image for `x86-64` on `15.1`. The goal was a usable image for developers. Enabling more architectures or updating the base image can be considered follow-up tasks.

Reply:
hm, ok. Let's try this and hope that @ggardet doesn't hate us for that :)

Reply:
`qemu-uefi-aarch64` is now built as part of ovmf package (cross compiled on x86), so it should be ok to keep it. Moreover this is _needed_ for aarch64...

Reply:
Let's try to revert this. hope this works now or we find a solution

Reply:
Please @ggardet @okurz take a look to this problem
![image](https://user-images.githubusercontent.com/11134265/99419625-9a44c580-28fc-11eb-89b4-43abf0826165.png)
https://build.opensuse.org/package/show/home:ilausuch:branches:devel:openQA/openQA_container_image_worker_x86

Reply:
In spite the code shouldn't be execute (I think) the OBS try to identify all the package before to do anything and determines that this doesn't exists in the repos. I cannot access to the logs of this build to see what happens, can you?

Reply:
I think we should add the following line in the PrjConf of the project:
```
# Workaround for OBS for Leap <= 15.2
%ifarch x86_64
Substitute: qemu-uefi-aarch64 qemu-ovmf-x86_64
%endif
%ifarch aarch64
Substitute: qemu-ovmf-x86_64 qemu-uefi-aarch64
%endif
```

Reply:
Worked!

Reply:
Ok, done